Magento_Review/web/js/go-to-review.babel.js

```diff
@@ -0,0 +1,38 @@
+define(() => (config, element) => {
+  element.addEventListener('click', () => {
+    const element = document.getElementById(config.reviewId);
+
+    if (element) {
+      const parentElement = findAncestor(element, 'tab__content');
+      setActiveFormTab(parentElement, parentElement.dataset.content);
+    }
+  });
+
```

Comment:
I would move content of anonymous to function to named function defined outside of the listener.

Reply:
moved, but don't know it's better than before